src/pds/ingress/service/sync_s3_metadata.py

```diff
@@ -288,7 +395,8 @@ def lambda_handler(event, context):
     parser = argparse.ArgumentParser(description="Invoke S3 metadata sync outside of Lambda.")
     parser.add_argument("bucket", help="S3 bucket name")
     parser.add_argument("prefix", help="S3 prefix")
+    parser.add_argument("--batch-size", type=int, default=1000, help="Batch size for processing (default: 1000)")
```

Comment:
Avoid repetition by changing the help text to
```python
help="Batch size for processing (default: %(default)d)"
```
(although "1000" does appear more places than just twice in this line 😉)